tcms/bugs/tests/test_admin.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+from django.urls import reverse
+from tcms.tests import LoggedInTestCase
+from tcms.utils.permissions import initiate_user_with_default_setups
+from tcms.tests.factories import TestRunFactory
+
+
+class TestBugAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+        cls.test_run = TestRunFactory()
```

Comment:
this is unused, remove it

Reply:
without this code (for authenticated user if I got it right), the test is failing on my local machine